nixos/tests/opentabletdriver.nix

```diff
@@ -20,8 +20,9 @@ import ./make-test-python.nix ( { pkgs, ... }: {
       machine.wait_for_x()
       machine.wait_for_unit("opentabletdriver.service", "alice")
 
-      machine.succeed("cat /etc/udev/rules.d/30-opentabletdriver.rules")
+      machine.succeed("cat /etc/udev/rules.d/99-opentabletdriver.rules")
       # Will fail if service is not running
-      machine.succeed("otd detect")
+      # TODO: Failing since v0.5.0, needs to investigate
+      # machine.succeed("otd --version")
```

Comment:
If someone is interested to investigate, this is the error:

```
machine # Unhandled exception. System.UnauthorizedAccessException: Could not connect to the pipe because it was not owned by the currentuser.
machine #    at System.IO.Pipes.NamedPipeClientStream.ValidateRemotePipeUser(SafePipeHandle handle)
machine #    at System.IO.Pipes.NamedPipeClientStream.TryConnect(Int32 timeout, CancellationToken cancellationToken)
machine #    at System.IO.Pipes.NamedPipeClientStream.ConnectInternal(Int32 timeout, CancellationToken cancellationToken, Int32 startTime)
machine #    at System.IO.Pipes.NamedPipeClientStream.<>c__DisplayClass20_0.<ConnectAsync>b__0()
machine #    at System.Threading.Tasks.Task.InnerInvoke()
machine #    at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
machine #    at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
machine # --- End of stack trace from previous location ---
machine #    at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
machine #    at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
machine # --- End of stack trace from previous location ---
machine #    at OpenTabletDriver.Desktop.RPC.RpcClient`1.Connect() in /build/source/OpenTabletDriver.Desktop/RPC/RpcClient.cs:line 25
machine #    at OpenTabletDriver.Console.Program.Main(String[] args) in /build/source/OpenTabletDriver.Console/Program.cs:line 26
machine #    at OpenTabletDriver.Console.Program.<Main>(String[] args)[   11.740550] otd-daemon[893]: Unhandled exception. System.UnauthorizedAccessException: Client connection (user id 0) was refused because it was not owned by the current user (id 1000).
machine #
machine # [   11.749911] otd-daemon[893]:    at System.IO.Pipes.NamedPipeServerStream.HandleAcceptedSocket(Socket acceptedSocket)
machine # [   11.755966] otd-daemon[893]:    at System.IO.Pipes.NamedPipeServerStream.<WaitForConnectionAsync>g__WaitForConnectionAsyncCore|24_0()
machine # [   11.760818] otd-daemon[893]:    at OpenTabletDriver.Desktop.RPC.RpcHost`1.Main() in /build/source/OpenTabletDriver.Desktop/RPC/RpcHost.cs:line 28
machine # [   11.766245] otd-daemon[893]:    at OpenTabletDriver.Daemon.Program.Main(String[] args) in /build/source/OpenTabletDriver.Daemon/Program.cs:line 63
machine # [   11.774522] otd-daemon[893]:    at OpenTabletDriver.Daemon.Program.<Main>(String[] args)
machine # [   11.798804] systemd[1]: Created slice system-systemd\x2dcoredump.slice.
machine # [   11.801136] systemd[1]: Started Process Core Dump (PID 936/UID 0).
machine # [   11.819781] systemd[1]: Started Process Core Dump (PID 935/UID 0).
machine # [   12.310822] systemd-coredump[937]: Cannot resolve systemd-coredump user. Proceeding to dump core as root: No such process
machine # [   12.312573] systemd-coredump[937]: Process 910 (OpenTabletDrive) of user 0 dumped core.
machine: output:
error:
machine # [   12.330083] systemd[1]: systemd-coredump@0-936-0.service: Succeeded.
Traceback (most recent call last):
  File "/nix/store/0fhfz90b5vc4k4gsk0gd6rj88m7gllz5-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 897, in run_tests
    exec(tests, globals())
  File "<string>", line 1, in <module>
  File "<string>", line 8, in <module>
  File "/nix/store/0fhfz90b5vc4k4gsk0gd6rj88m7gllz5-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 422, in succeed
    raise Exception(
Exception: command `otd --version` failed (exit code 134)
cleaning up
killing machine (pid 10)
(0.00 seconds)
builder for '/nix/store/4ap05p4fd7yhag4pajmmx40izklhn8p4-vm-test-run-opentabletdriver.drv' failed with exit code 1
error: build of '/nix/store/4ap05p4fd7yhag4pajmmx40izklhn8p4-vm-test-run-opentabletdriver.drv' failed
```

Reply:
The daemon now checks if the connection from a client is coming from the same user that started the daemon, so ensuring that the `otd` command is ran as `alice` did the trick.